src/index.html

```diff
@@ -13,6 +13,47 @@
     />
```

Comment:
HTML Formatting: It's a good practice to add an empty line between sibling blocks of code. It would be better to add an empty line between the `link` tag and the closing `head` tag to improve readability.